sdk/inc/azure/core/az_context.h

```diff
@@ -51,60 +47,41 @@ struct az_context
 #define _az_CONTEXT_MAX_EXPIRATION 0x7FFFFFFFFFFFFFFF
 
 /**
- * @brief The ultimate root of all #az_context instances. It allows you to cancel your entire
- * application. The #az_context_application never expires but you can explicitly cancel it by
- * passing its address to #az_context_cancel which effectively cancels all the #az_context child
- * nodes.
+ * @brief The application root #az_context instances. The #az_context_application never expires but
+ * you can explicitly cancel it by passing its address to #az_context_cancel which effectively
+ * cancels all the #az_context child nodes.
```

Comment:
nit: Move non-brief details to details. Add `()` for the method, and the` -> `its`

```suggestion
 * @brief The application root #az_context instances.
 * @details The #az_context_application never expires but you can explicitly cancel it by passing its address to 
 * #az_context_cancel() which effectively cancels all its #az_context child nodes.
```